### PR TITLE
[Summary data] Removing `AF_UNIX `and `AF_NETLINK` (showing only  `AF_INET `and `AF_INET6`)

### LINKED ISSUE #671: removing `AF_NETLINK` and `AF_UNIX` from summary data
## Reasons to remove these protocols.
(This is related to karmor summary network data bind points)
- Both protocols has no significance from security perspective. 
- It is unnecessarily complicating the app behaviour summary.

## Solution
- Only showing `AF_INET` and `AF_INET6` which is equivalent to **IPv4** and **IPv6**

### PR DESCRIPTION
**Pull request by yasin-cs-ko-ak**
This PR fixes this issue: #671

### CODE REVIEW

File: src/observability/summarizer.go [hunk 1]
```diff
@@ -40,14 +40,13 @@ func extractNetworkInfoFromSystemLog(netLog pb.Alert, pods []types.Pod, services
...
 		var path string
-		protocol = "AF_UNIX"
 		resslice := strings.Split(netLog.Resource, " ")
 		for _, locres := range resslice {
 			if strings.Contains(locres, "sun_path") {
 				path = strings.Split(locres, "=")[1]
 				if path != "" {
 					ip = path
-					bindAddress = path
+					protocol = "AF_UNIX"
```
Comment (nyrahul) [inline src/observability/summarizer.go:49]: @seswarrajan , do you understand why this change?

Comment (seswarrajan) [reply src/observability/summarizer.go:49]: No @nyrahul 
I've to skim through the code.

Comment (seswarrajan) [inline src/observability/summarizer.go:49]: @yasin-cs-ko-ak This does not seem like a fix. We need to fix/remove the code carefully so that AF_NETLINK/AF_UNIX shows for proper values.

**Review by seswarrajan — CHANGES_REQUESTED** [review]

File: src/observability/summarizer.go (hunk 1 above)
Comment (yasin-cs-ko-ak) [reply src/observability/summarizer.go:49]: @seswarrajan that value only shows for ingress/egress. In bind it is only AF_INET and AF_INET6. 
or we can change the AF_UNIX to UNIX as it is before.

Comment (yasin-cs-ko-ak) [reply src/observability/summarizer.go:49]: that AF_UNIX is not interfere with bind anymore.

Comment (seswarrajan) [reply src/observability/summarizer.go:49]: I understand that AF_UNIX and AF_NETLINK should not be present in the code itself and needs to be removed. 
We can discuss once.

**Review by seswarrajan — CHANGES_REQUESTED** [review]

File: src/observability/summary.go [hunk 2]
```diff
@@ -81,17 +81,19 @@ func GetSummaryData(request *opb.Request) (*opb.Response, error) {
...
-						Command:     loc_nw.Command,
-						IP:          loc_nw.PodSvcIP,
-						BindPort:    loc_nw.BindPort,
-						BindAddress: loc_nw.BindAddress,
-						Labels:      loc_nw.Labels,
-						Namespace:   loc_nw.Namespace,
-						Count:       strconv.Itoa(int(loc_nw.Count)),
-						UpdatedTime: loc_nw.UpdatedTime,
-					})
+					if strings.EqualFold(loc_nw.Protocol, "AF_INET6") || strings.EqualFold(loc_nw.Protocol, "AF_INET6") {
```
Comment (yasin-cs-ko-ak) [inline src/observability/summary.go:84]: @seswarrajan That's why I added here only to show inet and inet6. 

[L84](https://github.com/accuknox/discovery-engine/blob/3b5942ab845beec9ac9af0d526583c17e29aebe2/src/observability/summary.go#L84)

**Comment by yasin-cs-ko-ak** [comment]
@seswarrajan Kindly review it and approve it. If there's anything to discuss more, kindly let me know. Thanks

**Comment by seswarrajan** [comment]
@yasin-cs-ko-ak When this data is found in DE code, this needs to be completely ignored from processing. 
Not to be handled at all.

**Comment by yasin-cs-ko-ak** [comment]
> @yasin-cs-ko-ak When this data is found in DE code, this needs to be completely ignored from processing. 
> Not to be handled at all.

@seswarrajan if we remove this completely it won't shown for egress and ingress. We are only removing it for bind only right?
Any thoughts on that?

**Comment by seswarrajan** [comment]
Yes.. Bind only..
